config/defaults.js

```diff
@@ -122,5 +132,8 @@ module.exports = {
     password: process.env.TESTDROID_PASSWORD
   },
 
-  dockerWorkerPrivateKey: '/etc/docker-worker-priv.pem'
+  dockerWorkerPrivateKey: '/etc/docker-worker-priv.pem',
+
+  sslCertificate: '/etc/star_taskcluster-worker_net.crt',
+  sslKey: '/etc/star_taskcluster-worker_net.key'
```

Comment:
yea, I kept it separate specifically because it could probably be used later on.  I can move it to be 
```
ssl: {
  crtFile...
}